README.md

```diff
@@ -89,3 +89,15 @@ tar czvf libcanard.tgz cov-int
 ```
 
 Then upload the resulting archive to Coverity.
+
+### Running flake8 for python code
+
+Flake8 is a tool for checking correctness and style of python code, [Flake 8 docs](http://flake8.pycqa.org/en/latest/).
+
+The setup for Flake8 is defined in `setup.cfg`.
+
+Flake8 should be run from the top level directory, ideally code should be reviewed to make sure there are no
+flake errors before it is merged.
+```
+libcanard$ flake8
```

Comment:
The command doesn't seem to make sense. Did you mean that `flake8` needs to be run from the root directory? perhaps it's best to spell it out?